src/app/shared/components/warnings/warnings.component.scss

```diff
@@ -4,7 +4,14 @@
   border-left: 2px solid $warn;
   padding: 12px;
   background: #fff;
+
+  ::ng-deep {
+    p:last-of-type {
+      margin-bottom: 0;
+    }
+  }
```

Comment:
Before:
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/6022408/183098878-68957379-2969-4308-a2fe-21f2ffd55f01.png">


After:
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/6022408/183098762-40e45b6d-15c9-413b-9e1c-bbc8a70767d0.png">

Reply:
👍

bonus: could we avoid ::ng-deep somehow?

Reply:
No, because the paragraph is the content of the comment and not part of the warnings component.